selfdrive/manager/process.py

```diff
@@ -37,7 +38,7 @@ def launcher(proc, name):
     cloudlog.bind(daemon=name)
 
     # exec the process
-    mod.main()
+    getattr(mod, 'main')()
```

Comment:
Why change this? Both cases will result in `AttributeError` if main doesn't exist

Reply:
mypy fails without this change. I agree they are functionally the same